packages/x64dbg.vm/x64dbg.vm.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
   <metadata>
     <id>x64dbg.vm</id>
-    <version>2021.05.08.20230418</version>
+    <version>2024.01.21.20240123</version>
```

Comment:
```suggestion
    <version>2024.01.06.20240123</version>
```

Reply:
I think this should match the snapshotDate

Reply:
it does match 2024-01-06_21-29 is the snapshot date, this is 2024.01.06

Reply:
like I suggested? :)

Reply:
Oh I see I read the diff wrong ahah. Will fix